src/transaction/settings.ts

```diff
@@ -17,9 +17,9 @@ function setTransactionFlags(
   txJSON: TransactionJSON,
   values: FormattedSettings
 ) {
-  const keys = Object.keys(values)
+  const keys = Object.keys(values).filter((key) => AccountFlagIndices[key] !== undefined)
   assert.ok(
-    keys.length === 1,
+    keys.length <= 1,
```

Comment:
`keys.length` will be `0` if the user is not trying to set or clear any AccountSet Flags (which are booleans). It will be `1` if the user is trying to set or clear one of the flags. If it is `2`, then we throw the error.

So `<= 1` is correct: `0` is OK since the user might be changing other settings (not flags), e.g. `domain`. And `2` should result in an error since only one `SetFlag` can be provided per tx.

Reply:
Oh yeah ok got it. 
Thanks for clarifying :)